README.md

```diff
@@ -85,20 +127,28 @@ fish
 
 Alternatively, you can browse the same URL in a web browser, which should prompt you to download the binary file.
 
-### Configure local store directory
+### Check the status of an uploaded blob
 
-To configure the local storage directory, set `--storeDir` flag as an argument to the container.
-This will override the directy path used by Motion to store files locally.
-The path can be a [mounted volume](https://docs.docker.com/storage/volumes/), which then allows you to retain the stored files after the container
-is restarted and restore them back. For example, the following command uses a directory named `store` in the current working directory mounted as a volume on Motion container at path `/store`:
+In addition to retrieving data for a blob, you can also check the status of its storage on Filecoin:
 
 ```shell
-docker run --rm -p 40080:40080 -v $PWD/store:/store ghcr.io/filecoin-project/motion:main --storeDir=/store
+curl http://localhost:40080/v0/blob/ad7ef987-a932-495c-aa0c-7ffcabeda45f/status | jq .
```

Comment:
this example will lead to support questions around it not working. need to be clear to users how to get a blob ID from the previous call.

Reply:
it's pretty similar to retrieve example above no?